definitions/infra-awselasticsearchnode/summary_metrics.yml

```diff
@@ -6,14 +6,21 @@ providerAccountName:
 readThroughput:
   query:
     eventId: entityGuid
-    select: (average(`provider.IndexingRate.Average`)) / 60
+    select: (average(`provider.IndexingRate.Average`))
     from: DatastoreSample
-  unit: REQUESTS_PER_SECOND
+  unit: REQUESTS_PER_MINUTE
```

Comment:
This is not a valid unit. Please check [our docs](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/summary_metrics.md#metric-unit)